barretenberg/ts/src/barretenberg_cmds_wasm/index.ts

```diff
@@ -0,0 +1,75 @@
+import { Barretenberg } from '../barretenberg/index.js';
+import { BarretenbergCmds, Circuit, FieldsAndBinary } from '../barretenberg_cmds/index.js';
+import { Crs } from '../crs/index.js';
+import { Ptr } from '../types/ptr.js';
+import { RawBuffer } from '../types/raw_buffer.js';
+
+// This is the number of bytes in a UltraPlonk proof minus the public inputs.
+const numBytesInProofWithoutPublicInputs = 2144;
```

Comment:
I copied this blindly from the noir backend js, but I doubt it'll hold for all cases as we add more proving systems. Is there a way to get this info from the bb output?

Reply:
This is a hack so we didn't need to pass in the number of public inputs. If you've got the vk then you can extract the number of public inputs from that which will avoid the need for this.